src/main/scala/rocket/NBDcache.scala

```diff
@@ -669,7 +670,8 @@ class DataArray(implicit p: Parameters) extends L1HellaCacheModule()(p) {
 }
 
 class NonBlockingDCache(hartid: Int)(implicit p: Parameters) extends HellaCache(hartid)(p) {
-  override lazy val module = new NonBlockingDCacheModule(this) 
+  override lazy val module = new NonBlockingDCacheModule(this)
+  override def getOMSRAMS(): Seq[OMSRAM] = Nil
```

Comment:
Should this actually be `Nil`? Does a NonBlockingDCache really have no SRAMs within it?

Reply:
It should not be `Nil`, of course, but in @derekpappas' defense I did suggest to him that handling the `NonBlockingDCache` in this manner was of low priority.

Reply:
Ah, I see. In that case, I'd request that we at least leave a comment here stating that this is just a dummy value and that we need to eventually fix it. Otherwise it's hard to tell the difference between "this is intentionally an empty list" from "this is being set to an empty list because we want to defer the work of setting it to the correct value".

Reply:
Yep.